### ISSUE TITLE
Support for Multinomial Naive Bayes

### ISSUE THREAD

**Issue by krishnateja614** [jpmml/jpmml-sklearn#20]
Is there any estimate on when we can expect PMML support for Multinomial naive bayes model in sklearn?

**Comment by vruusmann** [comment]
What is the broader context of this feature request? The documentation for the `sklearn.naive_bayes.MultinomialNB` estimator type states that it's mostly suited for text classification purposes. However, the JPMML-SkLearn project doesn't address NLP workflows at the moment (eg. see issue #4), which means that there's no point in supporting the `MultinomialNB` estimator type as well.

**Comment by krishnateja614** [comment]
My problem consists of a column that takes many categorical variables (much
larger than the number sklearn random forest could handle natively which is
32 if I'm not wrong) and after one hot encoding the column, I was running
Multinomial Naïve Bayes which resulted a in very good accuracy compared to
other approaches. I guess this is similar to tf weighting where each
document contains only 1 word so a binary value instead of actual count.
Hence, I infact do not need any part of the NLP workflow like
countvectorizer or tfidf vectorizer but only Multinomial Naïve Bayes.

**Comment by KaidongYu** [comment]
@vruusmann I saw that this project support CountVectorizer now, is there any plan to support Multinomial Naive Bayes & Bernoulli Naive Bayes in sklearn?

**Comment by vruusmann** [comment]
@KaidongYu I'm definitely going to add proper support for the [`NaiveBayesModel` element](http://dmg.org/pmml/v4-3/NaiveBayes.html) into all my conversion libraries - JPMML-R, JPMML-SkLearn and JPMML-SparkML. I will be doing it all in one step, but I don't know when exactly do I have time for all that.

**Comment by Edzo0** [comment]
@vruusmann is it true that the Multinomial Naive Bayes si still not supported?
